miqa/settings.py

```diff
@@ -46,5 +49,25 @@ class ProductionConfiguration(MiqaMixin, ProductionBaseConfiguration):
     pass
 
 
+# TODO include HttpsMixin
+class DockerComposeProductionConfiguration(
+    MiqaMixin,
+    SmtpEmailMixin,
+    HttpsMixin,
+    _BaseConfiguration,
+):
+    """For the production deployment using docker-compose."""
+
+    @staticmethod
+    def before_binding(configuration: ComposedConfiguration) -> None:
+        # Register static files as templates so that the index.html built by the client is
+        # available as a template.
+        # This should be STATIC_ROOT, but that is bound as a property which cannot be evaluated
+        # at this point, so we make this assumption about staticfiles instead.
+        configuration.TEMPLATES[0]['DIRS'] += [
+            str(Path(configuration.BASE_DIR) / 'staticfiles'),
```

Comment:
`BASE_DIR` is already a `Path`, and this config value supports paths.

```suggestion
            configuration.BASE_DIR / 'staticfiles',
```